packages/design-system/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Same comment about `prettier` as with `tsconfig` -- I just personally don't know what can/can't be shared using this same type of approach.